src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -2,6 +2,69 @@
 
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        int salaryFirst = 0;
+        int salarySecond = 0;
+        int salaryThird = 0;
+        String firstName = "";
+        String secondName = "";
+        String thirdName = "";
+
+        String [] splitDataFrom = dateFrom.split("[.]");
```

Comment:
If you create a formatter, make it a constant field. It's recommended to define a DateTimeFormatter as a static final field if you are going to use it for parsing dates.